fcrepo-http-api/src/main/java/org/fcrepo/http/api/ContentExposingResource.java

```diff
@@ -591,6 +596,14 @@ protected void addResourceHttpHeaders(final FedoraResource resource) {
         addResourceHttpHeaders(resource, false);
     }
 
+
+    protected void addRegisterHeaders() {
+        if (fedoraPropsConfig.getFcrepoRegsiterApiBanner() == true) {
+            servletResponse.addHeader("USER-REGISTRATION", REGISTER_REPO_TEXT + " " + REGISTER_REPO_LINK);
```

Comment:
I personally think this is not a great idea as we are creating a new HTTP Header field, that carries a bunch of text on every response which is only human actionable (unlike the Link header which is machine actionable). Might be good to get some other's input at a Thursday call. 🤷

Reply:
Going to piggy back on this to ask if we really want additional text like this in the headers at all? There doesn't seem to be a good fit ([from the mdn reference](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers)), and they're more for the client/server rather than end user anyway. 

It seems to me that having the link header pointing to the registry should suffice so long as the `rel` describes it well enough.

Reply:
I tend to agree, but I presume the request came from the team somewhere when the jira ticket was created. Was there a reason at that time to do it this way?

Reply:
You would have to check with @dbernstein as this came out of discussions with the leadership group.